apps/backend/src/db/migrations/20260821120000_messages_revision.sql

```
@@ -0,0 +1,10 @@
+-- Revision 1 is the original body; every edit adds one. The number was until now
+-- derived as MAX(message_versions.version_number) + 1, which cannot be pinned by
+-- a reference without a second query per message. Backfill reproduces that
+-- derivation once, set-based, so historical rows agree with it.
+ALTER TABLE messages ADD COLUMN IF NOT EXISTS revision INTEGER NOT NULL DEFAULT 1;
+
+UPDATE messages m
+SET revision = v.max_version + 1
+FROM (SELECT message_id, MAX(version_number) AS max_version FROM message_versions GROUP BY message_id) v
```

SQLFluff (4.2.2)

[error] The 'GROUP' keyword should always start a new line.

(LT14)